Add Missing board file (#207)
betaflight/betaflight.com: docs/wiki/boards/Missing.md

> # Missing Board decription

Reviewer: ```suggestion
# Missing Board Description
```

> The Betaflight WIKI still haven't got a description for this board or version of board.

Reviewer: ```suggestion
The Betaflight WiKi does not have a description for this board.
```

Author: You have better English :-)

> You can try to find a previous board / board version from same Manufacturer at https://betaflight.com/docs/wiki/boards

Author: the bord.md files are found at https://betaflight.com/docs/wiki/boards/
the index to all board files are https://betaflight.com/docs/category/boards
not sure I understand, but seems to work

> Also you can request the Manufacturer to create this description as described at [https://betaflight.com/docs/manufacturer/manufacturer-design-guidelines](https://betaflight.com/docs/manufacturer/manufacturer-design-guidelines)

Reviewer: ```suggestion
You can ask the Manufacturer to add this description as described in [https://betaflight.com/docs/manufacturer/manufacturer-design-guidelines](https://betaflight.com/docs/manufacturer/manufacturer-design-guidelines)
```
